cpu/numactl.py

```diff
@@ -121,13 +121,18 @@ def setUp(self):
             self.input_file = self.params.get("input_file",
                                               default="/dev/zero")
         self.device = self.params.get('pci_device', default="")
-        if self.device:
-            if not os.path.isdir('/sys/bus/pci/devices/%s' % self.device):
-                self.cancel("%s not present in device path" % self.device)
-            self.cpu_path = "/sys/devices/system/node/has_cpu"
-            if not os.path.exists(self.cpu_path):
-                self.cancel("No NUMA nodes have CPU")
-            self.numa_dict = cpu.numa_nodes_with_assigned_cpus()
+
+        for test_ in ["preferred_node", "cpunode_with_membind", "physical_cpu_bind", "numa_pci_bind"]:
```

Comment:
is the variable name test_ fine with pylint ? Or is it possible to have something more descriptive of its use here ?

Reply:
Updated test_ -> subtest